coveralls.sh

```diff
@@ -7,16 +7,6 @@ DIR_SOURCE="$(find . -maxdepth 10 -type f -not -path '*/vendor*' -name '*.go' |
 BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
 echo "Using branch $BRANCH"
 
-pattern="refactor_*"
```

Comment:
As I wrote earlier, we can say that any `R4M` into master that touches `crypto` needs to get the `crypto`-part merged first.